backends-clickhouse/src/main/scala/io/glutenproject/execution/CHBroadcastBuildSideRDD.scala

```diff
@@ -16,14 +16,10 @@
  */
 package io.glutenproject.execution
 
-import java.util
+import java.util.concurrent.ConcurrentLinkedQueue
 
 object CHBroadcastBuildSideRDD {
 
   // Use for controling to build bhj hash table once.
-  val buildSideRelationCache = new util.LinkedHashMap[String, Long]() {
-    override def removeEldestEntry(eldest: util.Map.Entry[String, Long]): Boolean = {
-      size() > 100;
-    }
-  }
+  val buildSideRelationCache = new ConcurrentLinkedQueue[String]()
```

Comment:
Why Not ConcurrentMap? what's the bigo of `ConcurrentLinkedQueue.contains`

Reply:
it only needs hold a build table id, not need a (key, value) form.

Reply:
Maybe  `ConcurrentHashMap.newKeySet()` ?

Reply:
it also needs to remove the oldest build hash table id after inserting.

Reply:
OK